components/orchestration/cmd/serve.go

```diff
@@ -42,7 +42,7 @@ func newServeCommand() *cobra.Command {
 				fx.Invoke(func(lifecycle fx.Lifecycle, db *bun.DB) {
 					lifecycle.Append(fx.Hook{
 						OnStart: func(ctx context.Context) error {
-							return storage.Migrate(db, viper.GetBool(service.DebugFlag))
+							return storage.Migrate(db)
```

Comment:
Why removing the debug flag ?

Reply:
I said nothing, it was for goose, which is removed now

Reply:
Goose support this, i have not reimplemented it.
Do you think I should?

Reply:
It depends if we rely on it to debug migrations at first, but in any case I would say yes we should reimplement it (maybe later if needed if you're block for other things)